tests/MongoDB.Driver.Legacy.Tests/MongoCollectionTests.cs

```diff
@@ -3504,6 +3528,12 @@ public void TestWithWriteConcern()
         }
 
         // private methods
+        private bool AreSessionsSupported(IMongoClient client)
+        {
+            var description = client.Cluster.Description;
+            return description.LogicalSessionTimeout.HasValue;
```

Comment:
Surprisingly returns true for standalones also...

Reply:
Ack